openedx/frontend-lib-content-components
# feat: add labels and blockquotes to clear format

JIRA Ticket: [TNL-10475](https://2u-internal.atlassian.net/browse/TNL-10475)

Previously the "clear format" button in the tinyMCE editors did not include block quotes. With the addition of labels in the problem editor, this introduced another format that was not clear with the "clear format" button. Now, when the text is select both of these formats will be cleared.

Testing:

Remove block quote
1. Add a line of text and make it a block quote. Add a line of text that isn't a block quote. Add another block quote.
2. Highlight the one of the block quotes and click the "clear format" button.
3. Only the highlighted block quote's formatting is removed.

Remove label
1. Add a line of text and make it a label. Add a line of text that isn't a label. Add another label.
2. Highlight the one of the labels and click the "clear format" button.
3. Only the highlighted label's formatting is removed.

Remove label and block quote
1.  Add a line of text and make it a block quote.
2. Select some of the text inside the block quote and make it a label.
3. Highlight the label and click the "clear format" button.
4. Both the block quote and label are removed.
5. Undo change.
6. Select a non-label part of the blockquote and click the "clear format" button.
7. Only the block quote formatting is removed. The label format should still be applied.

Additionally ,this PR adds the function `setAssetsToStatic` when the problem editor is saved so images can be persisted in course re-runs. This capability has been previously tested with the Text Editor.
